app/src/main/AndroidManifest.xml

```diff
@@ -8,12 +8,12 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/Theme.ComposeColorPicker">
+        android:theme="@style/Theme.AppCompat.Light.NoActionBar">
         <activity
             android:name=".SampleColorPickerActivity"
             android:exported="true"
             android:label="@string/app_name"
-            android:theme="@style/Theme.ComposeColorPicker.NoActionBar">
+            android:theme="@style/Theme.AppCompat.Light.NoActionBar">
```

Comment:
Curious as to why this needed to change to support Desktop?

Reply:
It looks like I didn't add all the dependencies when I changed gradle to kts 😂, I can add them back if needed